layers/parameter_validation_utils.cpp

```diff
@@ -9294,6 +9294,45 @@ bool StatelessValidation::manual_PreCallValidateGetPhysicalDeviceSurfaceCapabili
         }
     }
 #endif
+
+    if (IsExtEnabled(device_extensions.vk_ext_surface_maintenance1)) {
+            const auto *surface_present_mode_compatibility =
```

Comment:
Minor indentation issue? Surprised clang-format didn't complain.
```suggestion
        const auto *surface_present_mode_compatibility =
```

Reply:
Yeah, I thought that was weird too.  I fixed it while working Spencer's comments, seems OK now.